src/sentry/flags/providers.py

```diff
@@ -90,7 +110,7 @@ def handle_launchdarkly_event(
     return [
         {
             "action": handle_launchdarkly_actions(access["action"]),
-            "created_at": datetime.datetime.fromtimestamp(result["date"] / 1000.0, datetime.UTC),
+            "created_at": timestamp_to_datetime(result["date"]),
```

Comment:
```suggestion
            "created_at": timestamp_to_datetime(result["date"] / 1000.0),
```
Suggesting this for readability. And change the helper fx accordingly. Timestamps are usually in seconds. Do we have a reference saying this date is milliseconds?

Reply:
@michellewzhang found this during her hands on testing with LaunchDarkly.  If there's explicit documentation for it she might be able to link it.

Reply:
> Timestamps are usually in seconds.

There's no standardization around it.  Though when I think "Unix timestamp" I think seconds.  That's clearly not the case for everyone as three of our providers choose to send timestamps in millisecond resolution.

Reply:
https://apidocs.launchdarkly.com/tag/Audit-log#operation/getAuditLogEntry

![image](https://github.com/user-attachments/assets/295f1d7e-b4d9-44a5-a41c-aaf56a2e49c2)

specified here that the time is in unix ms

Reply:
Thanks for the ref!
> milliseconds_to_datetime

Makes sense, thanks!